src/DIRAC/FrameworkSystem/Agent/ComponentSupervisionAgent.py

```diff
@@ -531,12 +532,18 @@ def checkURLs(self):
     gConfig.forceRefresh(fromMaster=True)
 
     # get port used for https based services
-    tornadoPortConfigPath = PathFinder.getComponentSection(
-        componentName=os.path.join('/Systems/Tornado'),
-        setup=self.setup,
-        componentCategory='Port'
-    )
-    self._tornadoPort = gConfig.getValue(tornadoPortConfigPath, self._tornadoPort)
+    try:
+      tornadoSystemInstance = PathFinder.getSystemInstance(
+          systemName="Tornado",
+          setup=self.setup,
+      )
+      self._tornadoPort = gConfig.getValue(
+          Path.cfgPath('/System/Tornado/', tornadoSystemInstance, "Port"),
+          self._tornadoPort,
+      )
+    except RuntimeError:
```

Comment:
Where does RuntimeError get raised and why?

Reply:
https://github.com/DIRACGrid/DIRAC/blob/rel-v7r2/src/DIRAC/ConfigurationSystem/Client/PathFinder.py#L34

Reply:
It gets raised in case there's no Tornado service installed, so that option is not in the CS.

Reply:
review OK?